### PR TITLE
Add Sdotbelow, dotbelowcomb to PriAfrican glyph set, update data.json mapping for PriAfrican glyph set

### LINKED ISSUE #141: Missing Sdotbelow Uppercase letter
The google font Latin_PriAfrican glyph set has a missing uppercase 'Sdotbelow' glyph

### PR DESCRIPTION
**Pull request by moyogo**
Fixes #141

### CODE REVIEW

File: Lib/glyphsets/data.json [hunk 1]
```diff
@@ -16432,7 +16432,8 @@
         "GF_Greek_Pro",
         "GF_Latin_Beyond",
         "GF_Latin_Vietnamese",
-        "GF_TransLatin_Arabic"
+        "GF_TransLatin_Arabic",
```
Comment (chrissimpkins) [inline Lib/glyphsets/data.json:16435]: @moyogo It looks like this is adding more data than just the PriAfrican updates here?  Can we make updates that are not PriAfrican specific in a separate PR?  I won't be the proper person to review those for inclusion here.

**Review by chrissimpkins — CHANGES_REQUESTED** [review]
Thanks Denis!  Do you mind opening a separate PR to integrate non-PriAfrican glyph set data in the data.json file?  OK to include the PriAfrican encoding tagging here, but I will not be the one to review the other glyph sets

File: Lib/glyphsets/data.json (hunk 1 above)
Comment (chrissimpkins) [reply Lib/glyphsets/data.json:16435]: This comment applies to a number of encodings in the changes to the data.json file proposed here.

File: GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist [hunk 2]
```diff
@@ -2,6 +2,46 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (chrissimpkins) [inline GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist:2]: It looks like this is the only plist file in the glyphs sub-directory.  Intentional? Or is this the incorrect path?

**Review by chrissimpkins — CHANGES_REQUESTED** [review]

File: Lib/glyphsets/data.json (hunk 1 above)
Comment (moyogo) [reply Lib/glyphsets/data.json:16435]: This only adds GF_Latin_PriAFrican.
JSON doesn’t have a comma on the line of the last element in a list, so adding a last element also changes the previous line.

File: GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist (hunk 2 above)
Comment (moyogo) [reply GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist:2]: I haven’t created this structure, I’m only following it. So far there has only been one file. It is the same in the other writing system directories.

The CustomFilter file has each glyphset separate anyway:
Screenshot 2023-11-07 at 11 24 00

File: Lib/glyphsets/data.json (hunk 1 above)
Comment (chrissimpkins) [reply Lib/glyphsets/data.json:16435]: Got it. Sorry I misinterpreted the diff

File: GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist (hunk 2 above)
Comment (chrissimpkins) [reply GF_glyphsets/Latin/glyphs/CustomFilter_GF_Latin.plist:2]: Copy that.  Ty!

**Review by chrissimpkins — APPROVED** [review]

**Comment by chrissimpkins** [comment]
Thank you very much for these changes Denis!

**Pull request merged**